README.md

```diff
@@ -59,17 +59,19 @@ See the command line options with:
    
 ### DNS records
 
-The feed-dns controller assumes that it can overwrite any entry in the supplied DNS zone and manages ALIAS
+The feed-dns controller assumes that it can overwrite any entry in the supplied DNS zone and manages ALIAS and CNAME
 records per ingress.
 
 On startup, all ingress entries are queried and compared to all the Record Sets in the configured hosted zone.
 
-Any ALIAS records pointing to one of the ELBs associated with this controller that do not have an ingress
-entry are deleted. For any new ingress entry, an ALIAS record is created to point to the correct ELB. Existing
+Any records pointing to one of the endpoints associated with this controller that do not have an ingress
+entry are deleted. For any new ingress entry, a record is created to point to the correct endpoint. Existing
 records which do not meet these conditions remain untouched.
 
-Each ingress must have the following tag `sky.uk/frontend-elb-scheme` set to `internal` or `internet-facing` so the A
-record can be set to the correct ELB.
+Each ingress must have the following tag `sky.uk/frontend-scheme` (`sky.uk/frontend-elb-scheme` is **deprecated**) set to `internal` or `internet-facing` so the
+record can be set to the correct endpoint.
```

Comment:
I thought internal/external were purely an  ELB concept. Do we need to extend it generically? Can't we just point to a single IP for the cname case?

Reply:
My understanding is that the internal is for anyone in corp to freely hit services using abc.bskyb.com/x but the external only accepts traffic from Akamai IPs; where Akamai also maps p.sky.com/abc/x/y/z back to abc.bskyb.com/x.

Reply:
It still applies if you have other internal services that need a more direct route and your external needs to be fronted by a CDN for example.

We could however make them optional i.e. have one or the other (or both), would that make more sense?